### PR TITLE
Initiate ritual script

### PR DESCRIPTION
**Pull request by KPrasch**
```
ape run initiate_ritual --network polygon:mumbai:infura -r ./deployment/artifacts/lynx.json
```

### CODE REVIEW

File: scripts/lynx/initiate_ritual.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+from deployment.params import Transactor
+from deployment.registry import contracts_from_registry
+from deployment.utils import check_plugins
+
+REGISTRY_FILEPATH = ARTIFACTS_DIR / "lynx.json"
+ONE_DAY = 60 * 60 * 24
+
+
+def main():
+    #
```
Comment (KPrasch) [inline scripts/lynx/initiate_ritual.py:15]:
```suggestion
    """
    Run this command with ape:
    ape run lynx initiate_ritual --network polygon:mumbai:<polygon-mumbai rpc endpoint>|infura
    """
```

File: scripts/lynx/initiate_ritual.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+def main():
+    """
+    Run this command with ape:
+    ape run lynx initiate_ritual --network polygon:mumbai:<polygon-mumbai rpc endpoint>|infura
+    """
+    check_plugins()
+    transactor = Transactor()
+
+    deployments = contracts_from_registry(filepath=REGISTRY_FILEPATH)
```
Comment (derekpierre) [inline scripts/lynx/initiate_ritual.py:22]: `contracts_from_registry` now takes a chain_id - https://github.com/nucypher/nucypher-contracts/blob/main/deployment/registry.py#L233. We'll need to specify the mumbai chain id in this call.

Comment (KPrasch) [reply scripts/lynx/initiate_ritual.py]: ✔️

File: scripts/initiate_ritual.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    print(f"Using account: {account}")
+    transactor = Transactor(account=account)
+
+    chain_id = project.chain_manager.chain_id
+    deployments = contracts_from_registry(filepath=registry_filepath, chain_id=chain_id)
+    coordinator = deployments[project.Coordinator.contract_type.name]
+
+    global_allow_list = deployments[project.GlobalAllowList.contract_type.name]
+    authority = transactor.get_account().address
+    providers = list(sorted(LYNX_NODES.keys()))
```
Comment (derekpierre) [inline scripts/initiate_ritual.py:48]: @KPrasch Careful here ... you may need to to take domain as a parameter...

**Pull request merged**